frontend/next.config.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
 
-const withNextIntl = require("next-intl/plugin")("./src/i18n/server.ts");
+const withNextIntl = require("next-intl/plugin")();
```

Comment:
see https://next-intl-docs.vercel.app/docs/usage/configuration - using a request.ts file, config will get picked up automatically. Not sure why this was in `server.ts` instead